_posts/newsletter/2020-04-09-this-month-in-solid.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-04-09
+permalink: /newsletter/2020-04-09
+tags: [monthly, updates]
+categories: [Updates]
+---
+{% include twis-intro.md %}
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+A big thank you to Travis Vachon, David Faveris, and Astrid Gamoneda for presenting at the first Solid World last Thursday. 
+
+In the first talk, [Travis Vachon](https://github.com/travis) from the USA presented a Solid app called [Concept](https://github.com/travis/concept). Concept is a Google docs equivalent that allows users to control their data. Travis built the concept with the aim to allow grassroots campaigns to organise themselves. Patch and WebSockets allowed for rudimentary collaborative editing and now wants to focus on embedding calendars and spreadsheets. 
+
+In the second talk, [David Faveris](https://forum.solidproject.org/u/Smag0) from France presented a Solid app called [Solidarity](https://github.com/scenaristeur/solidarity). Solidarity is a chat app built using the Shighl library, which fills the gap between the Solidarity UI and Solid Pods. David worked closely with designer Julian Cole who has been working with Shighl to allow for easy installation and minimal dependencies while working with linked data. 
+
+In the third talk, [Astrid Gamoneda Arrunada](https://github.com/empathyco/solid-pods-management/commits?author=astrd) from Spain presented a Solid app called [OhMyPod!](https://ohmypod.netlify.com/login)  developed by [Empathy](https://www.empathy.co/). OhMyPod! has a file management system with the ability to add friends with personal profiles. OhMyPod! is built using Inrupt React Generator, LDflex library and solid-file-client library. Astrid also looked at rdflib, which is powerful but complicated and somewhat slow. 
```

Comment:
Might want to phrase that last bit as "she found…"

Reply:
:+1: fixed.